modules/37_industry/subsectors/bounds.gms

```diff
@@ -160,4 +160,25 @@ if (cm_CCS_steel ne 1 OR cm_IndCCSscen ne 1,
 );
 $endif.cm_subsec_model_steel
 
+*** Populate values for v37_demFeIndst to ease introduction of new variale.  Can
+*** be removed once the variable is established.
+loop ((t,regi,
+       sefe(entySe,entyFe),
+       fe2ppfEn(entyFe,in),
```

Comment:
may need adaptation to ue?

Reply:
In theory yes, but `testOneRegi` was not screaming bloody murder, so probably it is irrelevant now.